pkg/ignition/ignition_test.go

```diff
@@ -24,36 +24,38 @@ import (
 	"io/ioutil"
 	"os"
 
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
 	"kubevirt.io/client-go/api"
 
 	v1 "kubevirt.io/api/core/v1"
 )
 
+var tmpDir string
+
+var _ = BeforeSuite(func() {
+	var err error
+	tmpDir, err = ioutil.TempDir("", "ignitiontest")
+	Expect(err).ToNot(HaveOccurred())
+	err = SetLocalDirectory(tmpDir)
+	if err != nil {
+		panic(err)
+	}
+})
+
+var _ = AfterSuite(func() {
+	os.RemoveAll(tmpDir)
+})
+
 var _ = Describe("Ignition", func() {
 
 	const vmName = "my-vm"
 	const namespace = "my-namespace"
-	var tmpDir string
+
 	// const ignitionLocalDir = "/var/run/libvirt/ignition-dir"
 	var vmi *v1.VirtualMachineInstance
 
-	BeforeSuite(func() {
```

Comment:
BeforeSuite must be called on top level, see please the error otherwise
```
It looks like you are trying to add a [BeforeSuite] node within a container
node.

BeforeSuite can only be called at the top level.

Learn more at:
http://onsi.github.io/ginkgo/#suite-setup-and-cleanup-beforesuite-and-aftersuite
```